common/tests/test_aws_service.py

```diff
@@ -1,22 +1,53 @@
+import base64
 import datetime
-from unittest import TestCase, mock
+from unittest import TestCase, mock, IsolatedAsyncioTestCase
```

Comment:
Same reason as below comment in test_copy_util.py , I believe IsolatedAsyncioTestCase is available in the later versions.

Reply:
It works for 3.8 and above
https://docs.python.org/3/library/unittest.html